content/references/rippled-api/api-conventions/base58-encodings.md

```diff
@@ -0,0 +1,31 @@
+# base58 Encodings
+
+The `rippled` APIs often use a [base58](https://en.wikipedia.org/wiki/Base58) encoding with a checksum (sometimes called "Base58Check") to represent [account addresses](accounts.html#addresses) and other types of values related to cryptographic keys. This encoding is the same as the one used for Bitcoin addresses, except that the XRP Ledger uses the following dictionary: `rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz`.
```

Comment:
It kind of shows up all over the place. The "See Also" section cover most of the cases that aren't address encoding. My thought was mostly about providing some context rather than listing everything.